### PR TITLE
feat/moar config

### PR DESCRIPTION
**Pull request by diasdavid**
for @lgierth <3

### CODE REVIEW

**Comment by diasdavid** [comment]
@libp2p/javascript-team can I get a quick CR so that @lgierth can spawn one of these? :)

File: src/signalling/bin.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+'use strict'
+
+const signalling = require('./index')
+const argv = require('minimist')(process.argv.slice(2))
```
Comment (dignifiedquire) [inline src/signalling/bin.js]: why are you using `minimist` and `yargs?

Comment (dignifiedquire) [inline src/signalling/bin.js]: Or rather why is minimist used here, when you added yargs to the dependencies

Comment (diasdavid) [inline src/signalling/bin.js]: ahaha that was me installing deps that came to mind right before boarding the plane, just in case xD

**Comment by diasdavid** [comment]
thank you for the review :)

there you go @lgierth :)

**Pull request merged**